src/main/java/Storage.java

```diff
@@ -0,0 +1,110 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private final Path databaseDirectoryPath;
+    private final Path databaseFilePath;
+
+    Storage(String databaseDirectoryPath) {
+        this.databaseDirectoryPath = Paths.get(databaseDirectoryPath);
+        this.databaseFilePath = Paths.get(databaseDirectoryPath + "/tasks.txt");
+    }
+
+    /**
+     * Loads the saved tasks of the user from /data/tasks.txt.
+     * If the directory does not exist, a new folder will be created.
+     *
+     * @return an ArrayList containing all the tasks (if any) of the user
```

Comment:
Should we put punctuation behind each parameter description? It might also be nice to start the parameter description with Capital letter:)